apps/condo/domains/user/components/UserPasswordResetButton.tsx

```diff
@@ -0,0 +1,127 @@
+import { Typography } from 'antd'
+import get from 'lodash/get'
+import React, { useEffect, useState } from 'react'
+import { useMutation } from '@core/next/apollo'
+import { useIntl } from '@core/next/intl'
+import { Button } from '@condo/domains/common/components/Button'
+import { runMutation } from '@condo/domains/common/utils/mutations.utils'
+import { useAuth } from '@core/next/auth'
+import { FormattedMessage } from 'react-intl'
+import { timer } from '../../common/utils/timer'
+import { START_PASSWORD_RECOVERY_MUTATION } from '../gql'
+import { extractRootDomain } from '@condo/domains/common/utils/url.utils.js'
+
+const PASSWORD_RESET_TIMEOUT = 60 // s
+
+const setCountDownDate = (date) => {
+    const domain = extractRootDomain(window.location.href)
+
+    document.cookie = `COUNTDOWN_DATE=${date}; path=/; domain=${domain}`
```

Comment:
it's better to use local storage. It's a front-end state and we don't need to know it on the server-side.

Reply:
Optional now, i guess. This is will be good solution if we will want to sync data with other devices.

Reply:
We also have a library for that